fit/accessFit.py

```diff
@@ -0,0 +1,81 @@
+from deephaven import DynamicTableWriter, Types as dht
```

Comment:
I would add the pup install here too so that it is just runable as a 1 thing

Reply:
sure, thankyou, I can see the advantage of simplicity of use over slowdown.